app/views/capacity.html

```diff
@@ -305,8 +305,8 @@ <h3 class="govuk-heading-m">Inactive & UPW cases</h3>
   <table id="outstandingReports" class="govuk-table data-table dataTable">
     <thead>
       <tr>
-        <th colspan="2">{{ childOrganisationLevelDisplayText }}</th>
-        <th class="" colspan="5">Inactive & UPW cases</th>
+        <th colspan="2" id='organisationLevel'>{{ childOrganisationLevelDisplayText }}</th>
+        <th class="" colspan="5" id='inactiveHeader' >Inactive & UPW cases</th>
```

Comment:
HTML attributes should be double quoted I think. I suppose we could do with an HTML linter